test/common/cdp-driver.js

```diff
@@ -295,7 +298,7 @@ CDP.New(options)
                                     if (unhandledExceptions.length === 0) {
                                         success(reply);
                                     } else {
-                                        fail(unhandledExceptions[0].exception.description);
+                                        fail(unhandledExceptions[0].exception.message);
```

Comment:
Oh, you figure it is the line breaks that break the protocol? Maybe we should still escape them here, in case the message also has multiple lines?